<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgColorSchemeSelector</class>
 <widget class="QWidget" name="WdgColorSchemeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="rbAuto">
     <property name="text">
      <string>Auto (follow system setting)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbLight">
     <property name="text">
      <string>Light</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbDark">
     <property name="text">
      <string>Dark</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
